shared/app/main-shared.native.js

```diff
@@ -60,11 +60,13 @@ class Main extends Component<any> {
     }
 
     return (
-      <RenderRoute
-        routeDef={this.props.routeDef}
-        routeState={this.props.routeState}
-        setRouteState={this.props.setRouteState}
-      />
+      <Fragment>
```

Comment:
needed?

Reply:
It is now that it's not being scoped to the route, the `GatewayDest` is a sibling to `RenderRoute`